go-controller/pkg/util/util.go

```diff
@@ -623,3 +635,23 @@ func ServiceFromEndpointSlice(eps *discovery.EndpointSlice, netInfo NetInfo) (*k
 
 	return &k8stypes.NamespacedName{Namespace: eps.Namespace, Name: svcName}, nil
 }
+
+// GetMirroredEndpointSlices retrieves all EndpointSlices in the given namespace that are managed
+// by the controller and are mirrored from the sourceName EndpointSlice.
+func GetMirroredEndpointSlices(controller, sourceName, namespace string, endpointSliceLister discoverylisters.EndpointSliceLister) (ret []*discovery.EndpointSlice, err error) {
+	mirrorEndpointSliceSelector := labels.Set(map[string]string{
+		discovery.LabelManagedBy: controller,
```

Comment:
how long can the value of controller here be?

Reply:
It is a constant currently, we manage it.